TCAT/Controllers/RouteOptionsViewController.swift

```diff
@@ -373,24 +352,6 @@ class RouteOptionsViewController: UIViewController {
                 routeSelection.updateSearchBarTitles(to: searchTo.name)
             }
 
-            // If don't have coordinates, fetch and restart process
-            if searchFrom.latitude == nil || searchFrom.longitude == nil {
```

Comment:
Are we sure we can take this out? Idk the answer but j checking

Reply:
Yeah I'm pretty confident we can. We had this in the past because when we had a `googlePlace`, it was in either one of two states: 1) we had its `placeIdentifier` but not its coordinates or 2) we had its `placeIdentifier` and its coordinates (which we obtained through `GooglePlacesDetails`. For MapKit, all `applePlaces` should be created w/ `lat` and `long` already and there isn't this two step process anymore. In terms of `busStops`, backend handles providing all coordinates for them